kotlin/android/app/src/main/java/dev/firezone/android/tunnel/TunnelManager.kt

```diff
@@ -200,8 +225,6 @@ internal class TunnelManager @Inject constructor(
             Log.d("Connlib","Attempting to load library from main app...")
             System.loadLibrary("connlib")
             Log.d("Connlib","Library loaded from main app!")
-            TunnelLogger.init()
-            Log.d("Connlib","Connlib Logger initialized!")
```

Comment:
We didn't actually need a separate function for this. I've removed it in favor of an `init_logging` function in the native-side `connect` entrypoint